airbyte-integrations/connectors/source-shopify/source_shopify/shopify_graphql/schema.py

```diff
@@ -21,6 +22,31 @@ class ARN(sgqlc.types.Scalar):
     __schema__ = shopify_schema
 
 
+class AbandonmentAbandonmentType(sgqlc.types.Enum):
```

Comment:
is this code auto generated?

Reply:
I updated the schema since it was the version of `2022-01` till this time. This did not affect anything, but I'm thinking of getting rid of this file in the future since it's huge to supply and manage. it's required for the present `products_graphql` stream, implemented by the contributor in `0.3.0`, and duplicates the logic of `products` stream,